### PR TITLE
fix: add anekemichael

### PR DESCRIPTION
**Pull request by anekemichael**
## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Review by anekemichael — COMMENTED** [review]
resolve #1309

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, 
Open your issue and use that issue no, instead of someone other.

If you need any help feel free to reach out.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by anekemichael** [comment]
Hey, Please how can i do that..?? I'm new here please

**Review by Pradumnasaraf — COMMENTED** [review]
Hey, you go on the issue section of this repo (https://github.com/EddieHubCommunity/awesome-github-profiles/issues) and create an issue, then add that to PR. 

Also, there is a full guide in the readme. 
Screenshot from 2022-08-11 23-12-06

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by MrKrishnaAgarwal — CHANGES_REQUESTED** [review]
**Create the issue [here](https://github.com/EddieHubCommunity/awesome-github-profiles/issues/new?assignees=&labels=&template=add-an-awesome-github-profile.md&title=Aneke-Michael)** and submit the correct `issue Id`

File: profiles/anekemichael.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/62024664/183684520-33c4059b-d4cc-4404-bfc9-9407370944e3.png",
+  "issueId": 66,
```
Comment (Panquesito7) [inline profiles/anekemichael.json:3]: This should be the correct issue ID: #1309.
```suggestion
  "issueId": 1309,
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by anekemichael** [comment]
Thanks so much

**Review by schmelto — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by anekemichael** [comment]
Thanks so much guys.. y'all are the best

**Review by MrKrishnaAgarwal — APPROVED** [review]
don't close it again and again
they'll merge the pr soon.

**Comment by anekemichael** [comment]
Ok.. Thanks

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by anekemichael** [comment]
Awesome, thanks!

**Comment by anekemichael** [comment]
Please, how can i add my profile to the profile.md page.?

**Pull request merged**